src/internal/group_api/mod.rs

```diff
@@ -136,6 +136,57 @@ impl GroupMetaResult {
     }
 }
 
+pub struct GroupCreateResult {
+    id: GroupId,
+    name: Option<GroupName>,
+    group_master_public_key: PublicKey,
+    is_admin: bool,
+    is_member: bool,
+    admins: Vec<UserId>,
+    members: Vec<UserId>,
+    created: DateTime<Utc>,
+    updated: DateTime<Utc>,
+    needs_rotation: Option<bool>,
+}
+impl GroupCreateResult {
```

Comment:
I assume this is mostly copy, but we should comment all of these instead of just some of them.

Reply:
Added comments to all